manuscript/draft.Rmd

```diff
@@ -166,3 +177,84 @@ None of the models are true forcasts as they use observed predictor variables (e
 * Need predictions at multiple time scales
 * Prediction quality at short time scales might not be a good indicator of 
   long-term effectiveness
+
+
+
+## Box 1: Ten simple rules for making and evaluating ecological forecasts
+
+### 1. Compare multiple modeling approaches
+
+Typically ecological forecasts use one modeling approach or a small number of
+related approaches. By fitting and evaluating multiple modeling approaches we
+can learn more rapidly about the best approaches for making predictions for a
+given ecological quantity. This includes comparing process based and data-driven
+models.
+
+### 2. Use time-series data when possible
+
+Forecasts describe how systems are expected to change through time. While some
```

Comment:
The current language does split them out "Using ecological and environmental time-series data allows..." I'm open to suggestions for additional changes, just not sure exactly what you're looking for. We want to keep things concise so that the box will fit on a single page, but I could see changes to "...allows the consideration of actual dynamics from both a process [@treddenick2016a] and error structure perspective." that might make the distinct importance of the two kinds of data clearer.